modules/programs/termite.nix

```diff
@@ -6,13 +6,6 @@ let
 
   cfg = config.programs.termite;
 
-  vteInitStr = ''
-    # See https://github.com/thestinger/termite#id1
-    if [[ $TERM == xterm-termite ]]; then
-      . ${pkgs.termite.vte-ng}/etc/profile.d/vte.sh
-    fi
```

Comment:
Is this compatible with `pkgs.vte`'s `vte.sh` script?

Reply:
This is the script I was using on Arch Linux on my previous configuration, I did not experience any issues. A quick look doesn't show any substantial difference between the two scripts.